packages/million/dom.ts

```diff
@@ -86,7 +86,7 @@ export const createEventListener = (
         el = el.parentNode;
       }
     });
-    document[EVENTS_REGISTRY].add(event);
```

Comment:
AFAIK, the polymorphic object is slow (per hidden class).

Do we have any benchmark for this change?